### ISSUE TITLE
Working Example of Anything and Not Just Images

### ISSUE THREAD

**Issue by wpfixit** [shrimpwagon/jquery-lazyloadanything#8]
The example you have here below is for images which is already widely used in LazyLoad.  

Can you provide an example of any HTML element using LazyLoad?  For example to LazyLoad a DIV with the class "lazy_me"?

$('img').lazyloadanything({
    'onLoad': function(e, LLobj) {
        var $img = LLobj.$element;
        var src = $img.attr('data-src');
        $img.attr('src', src);
    }
});

**Comment by shrimpwagon** [comment]
Haha, hey! I didn't think anyone was even using this....

So provide an example that doesn't have to do with images?

Sure np. I mean... all you do is just change the jquery selector

```javascript
$('#some-element').lazyloadanything();
```

**Comment by wpfixit** [comment]
Yes!  Would be great to see an example that is not an image but a DIV of content.

**Comment by shrimpwagon** [comment]
Do yall actually use it? I haven't touched this repo in years. Or did you just want to use it and curious?

**Comment by wpfixit** [comment]
Of curse I want to use it or I would not waste your time asking.

**Comment by shrimpwagon** [comment]
Hah, np! I'm glad to! Uh... give me a few mins today. I'll see what I can do.

**Comment by wpfixit** [comment]
Sweetness.  Will buy you a couple beers if this will work to LazyLoad any DIV by class.

**Comment by shrimpwagon** [comment]
Updated! I also added a funding page.

Do my a favor. Let me know how it works for newest Jquery version.

In theory should work find as this code is fairly simple.

Thank you!

**Comment by shrimpwagon** [comment]
ok try now. I fixed the funding page too.

**Comment by shrimpwagon** [comment]
If you are interested I have a Docker Laravel turn-key dev stack:

[https://github.com/CaneBayComputers/cbc-laravel](https://github.com/CaneBayComputers/cbc-laravel)

You can set up a bunch of websites easy.

**Comment by wpfixit** [comment]
I am sorry but I do not know what any of this means.  How can I find a working example where a DIV can be Lazy Loaded by class?

**Comment by shrimpwagon** [comment]
You don't lazy load a div per se. In your HTML there would already be an element. When the element comes into the viewport it then triggers the lazy load.
So, if you wanted to load a div another element that shows up in the viewport would have to trigger it.
So maybe something like this.....

You would have these elements already in the HTML:
```html
<div class="triggering-element"></div>
```
```javascript
$('.triggering-element').lazyloadanything({
    'onLoad': function(e, LLobj) {
        LLobj.$element.css('background-color', 'red');
});
```

**Comment by wpfixit** [comment]
Thanks but how would something like this be implemented on a web page.  A working example would be AMAZING!!!!
